src/components/Dashboard/TodoList.vue

```diff
@@ -751,13 +751,14 @@ export default class TodoList extends Mixins(DateMixin, EnumMixin, FilingMixin,
   }
 
   private expiresText (nameRequest: any): string {
```

Comment:
Logic change below! Note that `daysFromToday()` was updated accordingly.

It's more "pure" now. And it works no matter what the expiration date is. Examples:

Expires today at 12:01am Pacific (note that status comes from Namex API):
![image](https://user-images.githubusercontent.com/10002889/143333632-1a3ffcb0-f46e-45e5-afd4-18cc86f61b1b.png)

Expires today at 11:59pm Pacific:
![image](https://user-images.githubusercontent.com/10002889/143333708-898d3a07-e058-4628-b9a7-e759350d3021.png)

In both cases, the text "Expires today" is displayed correctly.